src/multiselect.js

```diff
@@ -0,0 +1,206 @@
+/**
+ * @license
+ * Copyright 2022 MIT
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Plugin main class.
+ */
+
+import * as Blockly from 'blockly/core';
+
+import * as ContextMenu from './multiselect_contextmenu';
+import * as Shortcut from './multiselect_shortcut';
+import {blockSelection, inMultipleSelectionMode,
+  hasSelectedParent, setBaseBlockDragger} from './global';
```

Comment:
Code style: Continuation lines in Google Javascript style are +4 from the previous indentation level.

Reply:
![image](https://user-images.githubusercontent.com/43995067/181035677-acecc4fb-1fd1-4a38-b654-cfcacb561921.png)

Actually I can see some warnings about this.